ui/src/components/Auth/LoginForm.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 The FoodUnit Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {inject, observer} from "mobx-react";
+import React from 'react';
+
+class LoginForm extends React.Component {
```

Comment:
`LoginForm` is my favorite component since it is very clean and obvious (explicit class, methods where they should be, simple `render` function etc.)